src/wp-includes/kses.php

```diff
@@ -2466,6 +2484,13 @@ function safecss_filter_attr( $css, $deprecated = '' ) {
 			}
 		}
 
+		if ( $found && $color_attr ) {
+			$css_value = trim( $parts[1] );
+			if ( preg_match( '/^rgb[a]?\((\d{1,3}%?),\s?(\d{1,3}%?),\s?(\d{1,3}%?)(,\s?|\s\/\s)(\d*(?:\.\d+)?)\)$/', $css_value ) ) {
```

Comment:
It might be possible to simplify the regex a lot here.

The goal of kses is to make the HTML (and CSS it contains) safe rather than valid. It's nice if it's valid but that's a problem for other parts of the code base.

You might be able to use `/^rgba?\([\s\d%\/.,]+\)$/` to limit the user to the allowed characters